app/download/views.py

```diff
@@ -77,6 +120,49 @@
     except ValueError:
         abort(404)
 
+    try:
+        check_scan_verdict(service_id, document_id, sending_method)
+    except (MaliciousContentError, SuspiciousContentError) as e:
+        current_app.logger.info(
+            "Malicious content detected, refused to download document: {}".format(e),
+            extra={
+                "service_id": service_id,
+                "document_id": document_id,
+            },
+        )
+        return jsonify(error=str(e)), MALICIOUS_CONTENT_ERROR_CODE
```

Comment:
Since this code returns the message to the user (vs. to the API), I think this CodeQL warning is now relevant.  

Wherever we are returning `jsonify(error=str(e))`, we should possibly return a fixed error message instead.

Reply:
Its internal though, so we are raising the error from the stack. I think if it was the API, I would agree on standardization. This only gets called through the API and returns a result that is user-friendly.

Reply:
Oh my bad, I thought the user was coming here directly, all good then!

Reply:
This actually is the endpoint that a user would hit if they receive a "link" attachment. It's a bit long but here is the code trail:
- API logic if "link" type attachment: https://github.com/cds-snc/notification-api/blob/main/app/delivery/send_to_providers.py#L209
- building the "URL" value that gets inserted in the email by the notification-api: https://github.com/cds-snc/notification-document-download-api/blob/main/app/upload/views.py#L60
- https://github.com/cds-snc/notification-document-download-api/blob/main/app/utils/urls.py#L21

that last link builds the `/d/<base64_uuid:service_id>/<base64_uuid:document_id>` url